src/ucp/wireup/wireup.c

```diff
@@ -2075,15 +2075,16 @@ double ucp_wireup_iface_lat_distance_v1(const ucp_worker_iface_t *wiface)
         wiface->attr.latency.c;
 }
 
-double ucp_wireup_iface_lat_distance_v2(const ucp_worker_iface_t *wiface)
+double ucp_wireup_iface_lat_distance_v2(const ucp_worker_iface_t *wiface,
```

Comment:
i would expect that wiface object will have enough information to know how many endpoints are expected to be created on it and whether it is prioritized, so passing prioritized_ep to this function should not be needed

Reply:
isn't the decision on how many UCT endpoints to create of each transport depends on the transport type (i.e RC or DC)?

Reply:
no, for example:
if EST_NUM_EPS=100 than:
1) RC should use NUM_EPS=100 for non prioritized EPs
2) RC should use NUM_EPS=20 for prioritized EPs